youtube_dl/YoutubeDL.py

```diff
@@ -848,6 +849,23 @@ def iter_playlistitems(format):
                 random.shuffle(entries)
 
             for i, entry in enumerate(entries, 1):
+                if self.params.get('playlistprompt', False):
+                    if 'title' in entry:
```

Comment:
Most playlists don't have titles in their entries' at this time thus it won't work for most of the cases.